documentation/chatmeleon-swagger.yaml

```diff
@@ -93,27 +90,29 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/Conversation"
+        "401":
+          description: Unauthorized - JWT token is missing or invalid.
+        "403":
+          description: Forbidden - User not allowed to retrieve the conversation.
```

Comment:
When would this happen?

Reply:
When maybe someone tries to retrieve the conversation that they are not part of I guess. I'm thinking there would be some mechanism to ensure that the person requesting the conversation and all of its metadata should be in the group and not an outsider

Reply:
>... there would be some mechanism to ensure that the person requesting the conversation and all of its metadata should be in the group and not an outsider

Yeah this is a must. Just that the doc is not too clear. Because, either 401 Unauthorized or 404 Not Found can be used for this (person requesting conversation they're not allowed to / a part of). We can decide upon which behavior and specify it in more details either in this Swagger doc or somewhere else.

Reply:
I was thinking that 401 would be invalidation of the token and 404 is for when the conversation is not found at all in the database but yea we need to talk and discuss the error codes again more thoroughly